### PR TITLE
(PDB-3159) Enable performance dashboard over https

### PR DESCRIPTION
**Pull request by florindragos**

### CODE REVIEW

**Comment by rbrw** [comment]
Hmm, this might need a rebase?  Wondering if it might have been started against main, but raised against 6.x (which is where I suspect we want it).

File: resources/public/dashboard/index.html [hunk 1]
```diff
@@ -91,6 +91,9 @@
 
 <script src="d3.v5.js"></script>
 <script src="charts.js"></script>
+<!-- token needed here ? -->
+<!-- <script src="d3.v5.js?token="></script>
+<script src="charts.js?token="></script> -->
```
Comment (austb) [inline resources/public/dashboard/index.html:96]: Just want to make sure we remember to remove these before merging.

**Review by austb — CHANGES_REQUESTED** [review]
This looks good, once those comments are removed this should be ready to merge.

We have quite a few features that are PE-only, but the code exists in part or in whole in the open source product, so having the token parameter here seems fine. I also tested that the page works with the certificate-allowlist and everything looks to function as expected there as well.

File: resources/public/dashboard/index.html (hunk 1 above)
Comment (florindragos) [reply resources/public/dashboard/index.html]: got catch. I forgot about them 👍

**Review by austb — APPROVED** [review]

**Pull request merged**